examples/sveltekit/test/basic.test.ts

```diff
@@ -0,0 +1,27 @@
+import { tick } from 'svelte'
+import { expect, test } from 'vitest'
+import Hello from '../src/lib/Hello.svelte'
+
+test('mount component', async() => {
+  const host = document.createElement('div')
+  host.setAttribute('id', 'host')
+  document.body.appendChild(host)
+  const instance = new Hello({ target: host, props: { count: 4 } })
+  expect(instance).toBeTruthy()
+  expect(host.innerHTML).toContain('4 x 2 = 8')
+  expect(host.innerHTML).toMatchSnapshot()
```

Comment:
Maybe it's better to replace it with this:

```ts
expect(host).toMatchSnapshot()
```

This way generated snapshot will have nice formatting.

Reply:
Done